src/chrome/content/scaffold/scaffold.js

```diff
@@ -303,6 +303,208 @@ var Scaffold = new function() {
 	});
 
 	/*
+	 * add template code
+	 */
+	this.addTemplate = Zotero.Promise.coroutine(function* (template, second) {
+		var cursorPos = _editors["code"].getSession().selection.getCursor();
+		var value = "";
+		switch(template) {
+			case "templateNewWeb":
+				value = `/*
```

Comment:
We should move these into separate files under `src/templates` or something like that and load them with `Zotero.File.getContentsAsync()`

Reply:
Yes, I wanted to do something like this, but didn't how exactly. Thanks for the hint.

How can I then specify the correct argument in `Zotero.File.getContentsAsync()`? A relative path didn't work for me.

Reply:
Okay, I figured it out: the easiest is to put this files into a subfolder of `src/chrome/content` and then address them by `chrome://scaffold/content/`. Moreover, I used `Zotero.File.getContentsFromURL` because I had problems with your suggested function.